config/site_settings.yml

```diff
@@ -1811,6 +1811,14 @@ user_preferences:
   default_categories_watching_first_post:
     type: category_list
     default: ''
+  
+  default_font_size:
+    type: enum
+    default: normal
+    choices:
+      - normal
+      - larger
+      - largest
```

Comment:
I'm just thinking out loud here but will this allow us room for expansion if we're already at the `largest`? I was thinking if using the actual pixel size as the name will be better since it is more intuitive to think in terms of the number of pixels instead of thinking about "how large is large".

Reply:
nahhh we need to keep the names simple here, I think this is fine for now.

Reply:
I wanted to avoid hard-coding pixel values, because they can easily be changed by a site using some basic CSS. If we decide to change this later it would be a trivial migration to change the names.